### PR TITLE
FIxes node access errors

### PR DESCRIPTION
**Pull request by itsjoekent**
This one
screen shot 2015-12-03 at 2 05 06 pm

### CODE REVIEW

File: lib/modules/dosomething/dosomething_global/dosomething_global.module [hunk 1]
```diff
@@ -135,7 +135,7 @@ function dosomething_global_node_access($node, $op, $account) {
   if (in_array($current_prefix, dosomething_global_get_countries())) {
     $current_lang = dosomething_global_convert_country_to_language($current_prefix);
     // Note: this cannot be strict === because it will fail, even if the value is zero.
-    if ($node->translations->data[$current_lang]['status'] == 0) {
+    if (isset($node->translations->data[$current_lang]) && $node->translations->data[$current_lang]['status'] == 0) {
```
Comment (sergiitk) [inline lib/modules/dosomething/dosomething_global/dosomething_global.module:138]: You can replace both statements with `if (empty($node->translations->data[$current_lang]))`;

Comment (itsjoekent) [inline lib/modules/dosomething/dosomething_global/dosomething_global.module:138]: we need to check the status though? its existence doesnt prove its published

Comment (sergiitk) [inline lib/modules/dosomething/dosomething_global/dosomething_global.module:138]: Oops, sorry.

**Comment by sergiitk** [comment]
:+1:

**Pull request merged**